src/module.c

```diff
@@ -5905,11 +5909,7 @@ RedisModuleCallReply *RM_Call(RedisModuleCtx *ctx, const char *cmdname, const ch
         if (!(flags & REDISMODULE_ARGV_NO_REPLICAS))
             call_flags |= CMD_CALL_PROPAGATE_REPL;
     }
-    /* Set server.current_client */
-    client *old_client = server.current_client;
-    server.current_client = c;
```

Comment:
@MeirShpilraien @guybe7 i'd like you to comment on this change.
do you you think we should keep the original client, or refer to the module client?
see the PR top comment for details about this change

Reply:
I think it is fine, moreover I remembered there was more places that assume `server.current_client` is the actual client and not the fake client so I think this is actually wrong setting the `server.current_client` here (I need to scan the code and make sure what I am saying here is still true/relevant)

Reply:
AFAIC we can remove these lines (they were added to address an issue that was solved another way)

also, now we are more similar to scripting, which also doesn't change current_client